### DISCUSSION TITLE
`@storybook/nextjs@7.5.3` Critical dependency warning

### DISCUSSION THREAD

**Discussion by TheMikeyRoss** [storybookjs/storybook#24757] [Help] [answered]
I keep seeing this warning when running storybook for my team ui kit 
https://github.com/sikka-software/hawa

it's running on storybook v7.5.3 and next v14.0.1
![image](https://github.com/storybookjs/storybook/assets/97709651/6e14621d-492e-4932-a2cf-94674b480b9a)

The warning seems to be coming from `@storybook/nextjs`

Adding the following in my Storybook config doesn't silence the warning
```
features: {
    storyStoreV7: true,
},
```

I know it's just a warning and I can just mentally ignore but I just love a clean console on a page refresh 😁

**Comment by valentinpalkovic** [comment, marked as answer]
As soon as https://github.com/storybookjs/storybook/pull/24710 is released, the warning will not be shown anymore.

**Reply by TheMikeyRoss** [reply]
Awesome, thanks
